packages/ember-views/lib/mixins/view_support.js

```diff
@@ -620,7 +620,7 @@ export default Mixin.create({
     this._super(...arguments);
 
     if (!this.elementId && this.tagName !== '') {
-      this.elementId = guidFor(this);
```

Comment:
Renaming it in `CurlyComponentMananger.create` throws a `Changing a view's elementId after creation is not allowed` error. I don't quite understand the difference between doing it in `CurlyComponentMananger.create` vs where it is now in `view_support`. To me, both seem to be allowing `id` setting per instance. Which seems to be [what's expected anyways](https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L1480).